src/app/shared/pipes/currency-repux.pipe.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { CurrencyRepuxPipe } from './currency-repux.pipe';
+
+describe('CurrencyRepux', () => {
+  describe('#transform()', () => {
+    it('return properly formatted strings', () => {
+      const pipe = new CurrencyRepuxPipe();
+      expect(pipe.transform('10.0000000000000000015')).toBe('REPUX 10.000000000000000002');
```

Comment:
In fact, there will never be such a situation because, as you said, the maximum precision of the REPUX token is 18. When we get some value from the blockchain / metaindexer, it can not contain 19 (or more) decimal places.